docs/pages/deploy-a-cluster/helm-deployments.mdx

```diff
@@ -9,8 +9,10 @@ layout: tocless-doc
 These guides show you how to set up a full self-hosted Teleport deployment using
 our `teleport-cluster` Helm chart.
 
+- [Teleport Cluster](./helm-deployments/kubernetes-cluster.mdx): Running a non-HA Teleport cluster in a Kubernetes cluster. A
```

Comment:
Can we expand "non-HA" to be more friendly to those not familiar with the jargon?

Reply:
Also, this sentence lacks a subject.

Reply:
Please see updates @alexfornuto